DOCS/xml-fields.md

```diff
@@ -0,0 +1,177 @@
+
+For license exceptions, the same pattern is used, with exceptions stored in the [`src/exceptions/` directory](../src/exceptions).
+
+A corresponding "test text" file, with a plain text version of the license or exception, should be added to the [`test/simpleTestForGenerator/` directory](../test/simpleTestForGenerator). Unlike the XML files, the same directory is used for the test files for licenses as well as exceptions. This text file will be used by the CI system to confirm that the XML template actually matches the "test text" for each license and exception.
+
+### Whitespace and formatting
+
+We aren't picky about tabs vs. spaces, number of spaces for indenting, etc.
+
+At some point maybe we'll pick a standard and update the existing files, but we haven't yet, so use whatever you like.
+
+### Beginning and end matter
+
+Within the .xml file, the first and last lines of the file should be the following:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+
+. . .
+
+</SPDXLicenseCollection>
+```
+
+### `<license>` or `<exception>` tag
+
+The main tag used to define the license or exception is, unsurprisingly, **`<license>`** or **`<exception>`**. All of the remaining content will be enclosed within a `<license></license>` or `<exception></exception>` pair of tags.
+
+There are two mandatory attributes for every `<license>` and `<exception>` tag:
+* `licenseId`: the unique SPDX Identifier for the license; should be identical to the filename
+* `name`: the longer "real name" of the license
+
+There are two additional attributes which are optional, but highly encouraged:
+* `isOsiApproved` (for licenses, not exceptions): either "true" or "false" based on whether this license has been approved by the [Open Source Initiative](https://opensource.org/licenses/alphabetical)
+* `listVersionAdded`: in which release version of the SPDX License List was the license first added, e.g. "3.19"
+
+Finally, if the license ID has been deprecated, two additional attributes should be included:
+* `isDeprecated`: should be "true" if the license ID is deprecated
+* `deprecatedVersion`: in which release version of the SPDX License List was the license first marked as deprecated, e.g. "3.19"
+
+Note that the deprecated tags refer to whether the ID has been deprecated _by SPDX_ -- in other words, if the SPDX Legal Team no longer recommends that the identifier should be used. Licenses that have been described as "deprecated" or "superseded" by the _license author_ might no longer be recommended to use for new code, but their identifiers remain valid License IDs on the SPDX License List.
+
+### License metadata
+
+There are three optional first-level metadata fields that can be nested immediately under `<license>` or `<exception>`, each of which is optional:
+
+* **`<crossRefs>`**: Defines cross-references to URLs where the license can be found in the wild
```

Comment:
"in the wild" is a bit informal.
"the license can be found in use by an open source project" or similar maybe?